cep-20.2.md

```diff
@@ -0,0 +1,832 @@
+    # only the listed paths are relocated
+    binary_relocation: bool (defaults to true) | [glob]
+
+    # Allow linking against libraries that are not in the run requirements
+    # (was `missing_dso_whitelist`)
+    missing_dso_allowlist: [glob]
+
+    # Allow runpath / rpath to point to these locations outside of the environment
+    # (was `runpath_whitelist`)
+    rpath_allowlist: [glob]
+
+    # what to do when detecting overdepending
+    overdepending_behavior: OneOf<"ignore" | "error"> # (defaults to "error")
+
+    # what to do when detecting overlinking
+    overlinking_behavior: OneOf<"ignore" | "error"> # (defaults to "error")
+
+  # REMOVED:
+  # pre-link: string (was deprecated for a long time)
+  # Wether to include the recipe or not in the final package - should be specified on command line or other config file?
+  # include_recipe: bool (defaults to true)
+  # noarch_python: bool
+  # features: list
+  # msvc_compiler: str
+  # requires_features: dict
+  # provides_features: dict
+  # preferred_env: str
+  # preferred_env_executable_paths: list
+  # disable_pip: bool
+  # marked as "still experimental"
+  # pin_depends: Enum<"record" | "strict">
+  # overlinking_ignore_patterns: [glob]
+  # defaults to patchelf (only cudatoolkit is using `lief` for some reason)
+  # rpaths_patcher: None
+  # post-link: path
+  # pre-unlink: path
+  # pre-link: path
+```
+
+### Script section
+
+```yaml
+script:
+  # the interpreter to use for the script
+  interpreter: string  # defaults to bash on UNIX and cmd.exe on Windows
+  # the script environment. You can use Jinja to pass through environment variables
+  # with the `env` key (`${{ env.get("MYVAR") }}`).
+  env: {string: string}
+  # secrets that are set as env variables but never shown in the logs or the environment
+  secrets: [string]
+  # The file to use as the script. Automatically adds `bat` or `sh` to the filename
+  # on Windows or UNIX respectively (if no file extension is given).
+  file: string  # build.sh or build.bat
+  # A string or list of strings that is the script contents (mutually exclusive with `file`)
+  content: string | [string]
+```
+
+## Source section
+
+```yaml
+source: [SourceElement]
+```
+
+where the different source elements are defined as follows.
+
+### URL source
+
+```yaml
+# url pointing to the source tar.gz|zip|tar.bz2|... (this can be a list of mirrors that point to the same file)
+url: url | [url]
+# destination folder in work directory
+target_directory: path
+# rename the downloaded file to this name
+file_name: string
+# hash of the file
+sha256: hex string
+# legacy md5 sum of the file (test both, prefer sha256)
+md5: hex string
+# relative path from recipe file
+patches: [path]
+```
+
+### Local source
+
+A path can be either:
+
+- directory ("../bla")
+- a path to an archive ("../bla.tar.gz")
+- a path to a file ("../bla.txt")
+
+```yaml
+# file, absolute or relative to recipe file
+path: path
+# if there is a gitignore, adhere to it (or not)
```

Comment:
Can you elaborate here what this implies? That those files will _not_ be copied over?